client/question/components/LearnerViewDirective.js

```diff
@@ -661,6 +661,48 @@ tie.directive('learnerView', [function() {
         var questionWindowDiv =
             document.getElementsByClassName('tie-question-window')[0];
 
+        // Set the name of the hidden property and the change event for
+        // visibility
+        var hidden;
```

Comment:
This could have a better name -- it's not clear what it represents, and at first I thought it was a boolean specifying whether the page was hidden or not. That said, I think it would be clearer to get rid of this and just have an isDocumentHidden() function, and handle the browser-specific part there.

Reply:
Fixed! Called it "determineHiddenAttributeNameForBrowser".